wagtail/wagtailadmin/site_summary.py

```diff
@@ -21,7 +21,13 @@ class PagesSummaryItem(SummaryItem):
     template = 'wagtailadmin/home/site_summary_pages.html'
 
     def get_context(self):
+        # If there is a single site, link to the homepage of that site
+        # Otherwise, if there are multiple sites, link to the root page
+        single_site = Site.objects.count() == 1
+        root = self.request.site.root_page if single_site else None
```

Comment:
We shouldn't ever rely on request.site being well-defined within the admin - see #1807. Not a problem here, of course, since we can just do `Site.objects.get()` instead...